src/cloud_tasks/cli.py

```
@@ -626,252 +470,885 @@ async def list_running_instances_cmd(args: argparse.Namespace, config: Config) -
+
+    provider = config.provider
+    provider_config = config.get_provider_config(provider)
+    queue_name = provider_config.queue_name
+    event_queue_name = f"{queue_name}-events"
+
+    # Determine database file path
+    db_file = args.db_file if args.db_file else config.run.db_file
+    if not db_file:
+        db_file = f"{provider_config.job_id}.db"
+
+    logger.info(f"Using database file: {db_file}")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)